app/controllers/surveys_controller.rb

```diff
@@ -34,6 +34,12 @@ def send_survey
     head :ok
   end
 
+  def index
+    @surveys = Survey.page(params[:page]).reverse_order
```

Comment:
Yeah i'll change it to order by `created_at` since it's more reliable. And write a test for it.